### DISCUSSION TITLE
Irina Nazarova - Evil Martians

### DISCUSSION THREAD

**Discussion by hipstersmoothie** [devtools-fm/devtools.fm#166] [Announcements]
- https://evilmartians.com
- https://twitter.com/inazarova
- https://anycable.io
- https://imgproxy.net

## Topics 

- sustainable opensource + commercial
- leadership 

### Background 

- [x] You've been in the industry for quite awhile and bounced between a lot of domains. Where did you career start?
- [x] What was you first step into leadership?

### Evil Martians

- [x] What is evil martians? I've seen the name in many readmes
- [x] How does evil martians approach work differently than other consulting agencies?
- [x] What type of work can you hire evil martins to do? Who have been past clients?

### Leadership

- [x] What do you do to foster tech titans like  like [https://github.com/ai](https://t.co/tIQLMWuBp1) and [https://github.com/palkan](https://t.co/3spxv9sowS)?
- [ ] ~How does the OSS work balance with the consulting work? Are they different teams?~

### Open Source + COSS

- [x] Why bet on devtools so much? Has the commitment to open source made the other work easier?
- [x] You have some product offerings like imgproxy and any cable. You were personally involved in AnyCable. What does it do?
- [x] Was it always the plan to offer a pro version or did it come about naturally through your process?
- [x] What your strategy around COSS? It's awesome how you all use your consulting work as an incubator for dev tool business ideas

### Future

- [ ] What technology has you excited right now?
- [x] What do you think the future of open source is? Do you think commercial offerings are needed for sustainability?

**Comment by hipstersmoothie** [comment]
- https://postcss.org
- https://oklch.com/#70,0.1,284,100

**Comment by irinanazarova** [comment]
https://www.razomforukraine.org/

**Comment by just-be-dev** [comment]
- https://www.homerow.app/
- https://github.com/unjs/magicast
